src/com/nutrons/steamworks/RobotBootstrapper.java

```diff
@@ -31,37 +32,52 @@
   @Override
   protected void constructStreams() {
 
-    this.serial = new Serial(PACKET_LENGTH *2, PACKET_LENGTH);
-    this.vision = Vision.getInstance(serial.getDataStream());
+    //this.serial = new Serial(PACKET_LENGTH *2, PACKET_LENGTH);
```

Comment:
Once lydia fixes the exception on serial not found, we won't have to comment this out to run our tests.